app/routes/synonyms/route.tsx

```diff
@@ -10,12 +10,13 @@ import { Outlet } from '@remix-run/react'
 import { GridContainer } from '@trussworks/react-uswds'
 
 import { getUser } from '../_auth/user.server'
+import { feature } from '~/lib/env.server'
 
 export async function loader({ request }: LoaderFunctionArgs) {
   const user = await getUser(request)
   const isModerator = user?.groups.includes('gcn.nasa.gov/circular-moderator')
 
-  if (!isModerator) {
+  if (!isModerator || !feature('SYNONYMS')) {
```

Comment:
We can probably do without this feature flag check here because the synonyms interface is only visible to moderators.